docs/packages/compile-package.md

```diff
@@ -218,4 +240,112 @@ When the `bash onion_buildenv build_all_packages` command is run, the Onion Open
+make[3]: Entering directory '/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk/build_dir/target-mipsel_24kc_musl/c-example-1.0'
+mipsel-openwrt-linux-musl-gcc -Os -pipe -mno-branch-likely -mips32r2 -mtune=24kc -fno-caller-saves -fno-plt -fhonour-copts -msoft-float -ffile-prefix-map=/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk/build_dir/target-mipsel_24kc_musl/c-example-1.0=c-example-1.0 -mips16 -minterlink-mips16 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -DPIC -fpic -c main.c -o main.o
+main.c: In function 'main':
+main.c:18:46: error: expected ';' before 'curl'
+   18 |     printf("Fetching To Do from '%s'\n", url)
+      |                                              ^
+      |                                              ;
+......
+   21 |     curl = curl_easy_init();
+      |     ~~~~                                      
+make[3]: *** [makefile:24: main.o] Error 1
+make[3]: Leaving directory '/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk/build_dir/target-mipsel_24kc_musl/c-example-1.0'
+make[2]: *** [Makefile:52: /home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk/build_dir/target-mipsel_24kc_musl/c-example-1.0/.built] Error 2
+make[2]: Leaving directory '/home/ubuntu/Example-OpenWRT-Packages/c-example'
+time: package/feeds/custom/c-example/compile#0.14#0.26#0.31
+    ERROR: package/feeds/custom/c-example failed to build.
+make[1]: *** [package/Makefile:129: package/feeds/custom/c-example/compile] Error 1
+make[1]: Leaving directory '/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk'
+make: *** [/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk/include/toplevel.mk:225: package/c-example/compile] Error 2
+make: Leaving directory '/home/ubuntu/openwrt-sdk-wrapper/openwrt-sdk'
+\n>> ERROR: Compile of package c-example returned code 2
+\n>> PACKAGE COMPILATION ERROR!\n
+```
+
+## Rerun Package Compilation
+
+Re-compiling requires a re-run of the feed update step following any code or configuration file modifications, including changes to the listed repositories.
+
+These changes can include:
+
+1. Changing which repos are listed in the profile config file.
+2. Updates to the code in the repo - new versions of code available or fixes to issues causing compilation errors.
+
+Run the following command to ensure all changes are pulled in before compilation:
```

Comment:
Let's add a Header 3 here to separate the intro from the actual commands. Maybe something like "Commands to Rerun Compilation" ?